tests/LinAlg/matrixTests.hpp

```diff
@@ -504,11 +497,11 @@ class MatrixTests : public TestBase
   // {
```

Comment:
This entire test is still commented out. Should we be testing `matrixAddToSymDenseMatrixUpperTriangle`?